meson.build

```diff
@@ -576,6 +576,8 @@ if get_option('enable-x11')
     libxkbcommon_x11_internal = static_library(
```

Comment:
fwiw, I'd appreciate it if you could rename this to `libxkbcommon_x11_test_internal` or something, I had to double-check `meson.build` to ensure this wasn't used anywhere else. Having it in the name makes it more obvious IMO

Reply:
Done